packages/gatsby-plugin-mdx/gatsby-node.js

```diff
@@ -41,7 +41,7 @@ exports.preprocessSource = require(`./gatsby/preprocess-source`)
  */
 exports.onCreateBabelConfig = ({ actions }) => {
   actions.setBabelPlugin({
-    name: `@babel/plugin-proposal-object-rest-spread`,
+    name: require.resolve(`@babel/plugin-proposal-object-rest-spread`),
```

Comment:
Hmmm... maybe this is wrong, wonder if there’s a way to set the plugin this way. Might have to take a look at `createConfigItem`...

Reply:
Babel plugins can/should be absolute paths, so this looks fine I think 🤔

Reply:
But now we cannot dedup it with other part of the app importing the same babel plugin, can we?
Maybe babel handle it nicely internally, but I’m not sure 🤔.

Reply:
Looks like there are already some same [usages](https://github.com/gatsbyjs/gatsby/blob/93f0645dc0088ef114eb6ca7ed05f23cae3556ce/packages/gatsby-plugin-typescript/src/gatsby-node.js#L4-L16) like this, I think it's fine!